### PR TITLE
ADD:meeting protocol 2021-01-14

### LINKED ISSUE #63: Meeting-Protocol 2020-01-14

### PR DESCRIPTION
**Pull request by BlueEnni**
Please check the 'Demodata' & the 'To-Do' section - I may have missed some information on those two parts

### CODE REVIEW

File: documentation/meetings/2021-01-14.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+	- Backend
+		○ Tests von Dominik wurden eingerichtet
+		○ Die neue Datenbankstruktur wurde besprochen
+	    + 12 Tabellen (alle für das Portal)
+    	+ Die Tabellen tauchen auch wieder im ER-Modell auf
+	
+	  ○ seqdata (Demodatensatz)
+		  + rolle, team, membership (member & team …)
+	
+    ○ Route hat jetzt einen Scope
```
Comment (DominikRuf) [inline documentation/meetings/2021-01-14.md]:
```suggestion
    ○ Jedes Model hat jetzt einen DefaultScope und verschiedene passende Scopes
```

**Review by DominikRuf — CHANGES_REQUESTED** [review]
Kleinigkeit, die ich anders schreiben würde. Sonst top Mitschrieb, Danke! :+1:

File: documentation/meetings/2021-01-14.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+
+	- Backend
+		○ Tests von Dominik wurden eingerichtet
+		○ Die neue Datenbankstruktur wurde besprochen
+	    + 12 Tabellen (alle für das Portal)
+    	+ Die Tabellen tauchen auch wieder im ER-Modell auf
+	
+	  ○ seqdata (Demodatensatz)
+		  + rolle, team, membership (member & team …)
```
Comment (DominikRuf) [inline documentation/meetings/2021-01-14.md]:
```suggestion
		  + organization, user, role, team, membership (user & team), roleAssignment (user & role)
```

**Review by DominikRuf — CHANGES_REQUESTED** [review]

**Comment by BlueEnni** [comment]
Thx @DominikRuf  - added your suggestions

**Review by DominikRuf — APPROVED** [review]

**Pull request merged**